### DISCUSSION TITLE
Load js error in JMeter

### DISCUSSION THREAD

**Discussion by fit2-zhao** [mozilla/rhino#1348] [Q&A]
Loading the following JS in JMeter will report an error

var navigator = this;
var window = this;

g = new Packages.org.mozilla.javascript.tools.shell.Global(Packages.org.mozilla.javascript.Context.getCurrentContext());
this.load = g.load;
// js https://gist.github.com/tresf/f62aeb28d31d382fdb30cad11372d693#file-jsrsasign-all-min-js
load("/jsrsasign-all-min.js");

**Comment by p-bakker** [comment]
You haven't included what kind of error you're getting nor any analysis of what the specific issue with Rhino might be.
